Userland/Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.h

```diff
@@ -313,4 +318,22 @@ class ClampCalculationNode final : public CalculationNode {
     NonnullOwnPtr<CalculationNode> m_max_value;
 };
 
+class AbsCalculationNode final : public CalculationNode {
+public:
+    static ErrorOr<NonnullOwnPtr<AbsCalculationNode>> create(NonnullOwnPtr<CalculationNode>);
+    ~AbsCalculationNode();
+
+    virtual ErrorOr<String> to_string() const override;
+    virtual Optional<CalculatedStyleValue::ResolvedType> resolved_type() const override;
+    virtual bool contains_percentage() const override { return false; };
+    virtual CalculatedStyleValue::CalculationResult resolve(Optional<Length::ResolutionContext const&>, CalculatedStyleValue::PercentageBasis const&) const override;
+    virtual ErrorOr<void> for_each_child_node(Function<ErrorOr<void>(NonnullOwnPtr<CalculationNode>&)> const&) override;
+
+    virtual ErrorOr<void> dump(StringBuilder&, int indent) const override;
+
+private:
+    AbsCalculationNode(NonnullOwnPtr<CalculationNode>);
+    NonnullOwnPtr<CalculationNode> m_a;
```

Comment:
I know the spec calls this parameter "A", but please name it something else that's more explanatory without looking at the spec. (`m_value` would work.)